lib/elixir/lib/supervisor/spec.ex

```diff
@@ -97,11 +97,11 @@ defmodule Supervisor.Spec do
       to give the subtree enough time to shutdown; it can also be used with
       workers with care
 
-    * any integer - the value of `:shutdown` can also be any integer meaning
+    * any integer >= 0 - the amount of time in milliseconds
```

Comment:
This will not render nicely if without backticks. I'd much rather go with "any non-negative integer".